Web/View/Items/New.hs

```diff
@@ -19,37 +22,53 @@ renderForm :: Item -> [BidStep] -> Html
 renderForm item bidSteps = formFor item [hsx|
     {(textField #title)}
 
-    {forEachWithIndex bidSteps renderFormBidStep}
+    <div class="multiple-field-wrapper border p-4 my-4">
+
+        <div class="multiple-field-inner">
+            {forEachWithIndex bidSteps renderFormBidStep}
+
+        </div>
+
+
+
+        <div class="multiple-field-add btn btn-secondary mt-4">Add Another Bid Step</div>
+    </div>
 
     {submitButton}
 |]
 
 -- renderFormBidStep :: BidStep -> Html
 renderFormBidStep (index, bidStep) = [hsx|
-    <h3>Bid Step #{index + 1}</h3>
-    <div class="form-group">
-        <label>
-            Min
-        </label>
-        <input type="number" name="min" value="{get #min bidStep}" class={classes ["form-control", ("is-invalid", isInvalidMin)]}/>
-        {minFeedback}
-    </div>
+    <fieldset class="multiple-field border p-4 my-4">
+        <div class="d-flex flex-row justify-content-between">
+            <h3>Bid Step</h3>
+            <a class="multiple-field-remove" href="javascript:void(0)">X</a>
+        </div>
 
-    <div class="form-group">
-        <label>
-            Max
-        </label>
-        <input type="number" name="max" value="{get #max bidStep}" class={classes ["form-control", ("is-invalid", isInvalidMax)]}/>
-        {maxFeedback}
-    </div>
+        <div class="form-group">
+            <label>
+                Min
+            </label>
+            <input type="number" name="min" value="{get #min bidStep}" class={classes ["form-control", ("is-invalid", isInvalidMin)]}/>
+            {minFeedback}
+        </div>
 
-    <div class="form-group">
-        <label>
-            Step
-        </label>
-        <input type="number" name="max" value="{get #step bidStep}" class={classes ["form-control", ("is-invalid", isInvalidStep)]}/>
-        {stepFeedback}
-    </div>
+        <div class="form-group">
+            <label>
+                Max
+            </label>
+            <input type="number" name="max" value="{get #max bidStep}" class={classes ["form-control", ("is-invalid", isInvalidMax)]}/>
```

Comment:
To fix the console warnings try this:

```diff
-value="{get #max bidStep}"
+value={inputValue (get #max bidStep)}
```

Reply:
Yes the quotes need to be removed :)

Try this:
```html
value={if didChange #max bidStep
    then inputValue (get #max bidStep)
    else ""
}
```

Reply:
Yeah, the `if didChange` did the trick.